_layouts/home.html

```diff
@@ -32,7 +32,7 @@
 
         {% include google-tag-manager.html %}
 
-        <a href="https://github.com/gocd/gocd/">
+        <a href="https://github.com/gocd/gocd/" analytics-label="github-fork-me">
```

Comment:
Ok, that makes sense. If you use localStorage, I can see how you wouldn't need to bother with waiting for the action. I'm convinced. :)